app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PostsController < ApplicationController
+  def index
+    @posts = @user.posts.order(created_at: :asc)
+  end
+
+  def show
+    @post = @user.post.find(params[:id])
+  end
+
+  # def new
+  #   @post = Post.new
+  # end
+
+  # def create
+  #   @post = Post.new(post_params)
+  #   if @post.save
+  #     redirect_to @post
+  #   else
+  #     render 'new'
+  #   end
+  # end
+
+  # def post_params
+  #   params.require(:post).permit(:title, :text, :comments_counter, :likes_counter)
+  # end
```

Comment:
- [ ] @haftamudesta Its not god practice having commented code in your codebase , kindly consider removing commented code.